### PR TITLE
fix(Designer): Migrate some UI to Fluent v9 components

### PR DESCRIPTION
**Pull request by Eric-B-Wu**
We were having issues with darkmode, I'm not sure if this is limited to standalone env as it seems vscode may also have the issue. 

I'm migrating all Text componets to v9, as well as any other cases where I'm not seeing dark mode get caught

### CODE REVIEW

File: libs/designer-ui/src/lib/connectorsummarycard/index.tsx [hunk 1]
```diff
@@ -42,7 +49,13 @@ export const ConnectorSummaryCard = (props: ConnectorSummaryCardProps) => {
         />
       </div>
       {displayRuntimeInfo ? (
-        <div className="msla-connector-summary-labels">{isBuiltIn ? <Text className="msla-psuedo-badge">{category}</Text> : null}</div>
+        <div className="msla-connector-summary-labels">
+          {isBuiltIn ? (
```
Comment (rllyy97) [inline libs/designer-ui/src/lib/connectorsummarycard/index.tsx:53]: FV9 had an actual badge component we could convert this to

**Review by ccastrotrejo — APPROVED** [review]

File: libs/designer-ui/src/lib/html/plugins/toolbar/ColorPickerTextInput.tsx [hunk 2]
```diff
@@ -26,7 +27,12 @@ export function TextInput({ label, value, onChange, placeholder = '', 'data-test
 
   return (
     <div className="msla-colorpicker-input-wrapper">
-      <label className="msla-colorpicker-input-label">{label}</label>
+      <label
```
Comment (rllyy97) [inline libs/designer-ui/src/lib/html/plugins/toolbar/ColorPickerTextInput.tsx:30]: Can we convert this to a V9 component so it just colors passively?

File: libs/designer-ui/src/lib/html/plugins/toolbar/DropdownColorPicker.tsx [hunk 3]
```diff
@@ -97,7 +100,9 @@ export const DropdownColorPicker = ({
   return (
     <DropDown {...dropdownProps} disabled={disabled} stopCloseOnClickSelf buttonIconSrc={buttonIconSrc} editor={editor}>
       <div className="color-picker-wrapper" style={{ width: WIDTH }} ref={innerDivRef}>
-        <Text className="color-picker-title">{capitalizeFirstLetter(title ?? '')}</Text>
+        <Text className="color-picker-title" style={{ color: isInverted ? constants.INVERTED_TEXT_COLOR : constants.STANDARD_TEXT_COLOR }}>
```
Comment (rllyy97) [inline libs/designer-ui/src/lib/html/plugins/toolbar/DropdownColorPicker.tsx:103]: I'm wondering why these aren't coloring properly by default, do these html components render outside of the FluentProvider or something?

**Review by rllyy97 — APPROVED** [review]

File: libs/designer-ui/src/lib/html/plugins/toolbar/DropdownColorPicker.tsx (hunk 3 above)
Comment (Eric-B-Wu) [reply libs/designer-ui/src/lib/html/plugins/toolbar/DropdownColorPicker.tsx:103]: yeah they do, which was why these weren't getting caught

File: libs/designer-ui/src/lib/html/plugins/toolbar/ColorPickerTextInput.tsx (hunk 2 above)
Comment (Eric-B-Wu) [reply libs/designer-ui/src/lib/html/plugins/toolbar/ColorPickerTextInput.tsx:30]: Think this dropdown also gets rendered outside, so when using a label we won't be able to get the provider. However, Eric from PA is working/ has a PR for moving these dropdowns to fluent ui, so this won't be the case for very long

**Pull request merged**